tensorflow_datasets/core/download/downloader.py

```diff
@@ -148,21 +149,39 @@ def _sync_download(self, url, destination_path):
         'https': os.environ.get('TFDS_HTTPS_PROXY', None),
         'ftp': os.environ.get('TFDS_FTP_PROXY', None)
     }
+    CA_BUNDLE = os.environ.get(
+        'TFDS_CA_BUNDLE',
+        None) or os.environ.get(
+        'REQUESTS_CA_BUNDLE',
+        None)
+    if CA_BUNDLE:
+      CA_BUNDLE = extract_zipped_paths(CA_BUNDLE)
+    if not hasattr(ssl, '_create_unverified_context'):
+      # For python version <= 2.7.8
+      ssl.__dict__['_create_unverified_context'] = lambda: None
+      ssl.__dict__['create_default_context'] = lambda *args, **kwargs: None
+    CA_VERIFY = {
+        'urllib': ssl._create_unverified_context() if CA_BUNDLE is None or len(CA_BUNDLE) == 0
```

Comment:
"not X" takes care of X is None or len(X) == 0.

'urllib': ssl.create_default_context(CA_BUNDLE) if CA_BUNDLE else ssl_,create_unverified...